src/agent-page.js

```diff
@@ -1,16 +1,70 @@
 const agentElements = {
-  navButtons: `<button type="button" name="My Customers">
+  navButtons: `
+  <button type="button" id="my-customers" name="My Customers">
     <img src="./images/customers.svg" alt="">
-    My Customers
+    Pending Trips
   </button>
-  <button type="button" name="Destinations">
+  <button id="destinations" type="button" name="Destinations">
     <img src="./images/003-world.svg" alt="">
     Destinations
   </button>
   <button id="log-out-btn" type="button" name="Log Out">
     <img src="./images/login.svg" alt="">
     Log Out
   </button>`,
+  totalEarned: (earned) => {
+    return `<p>Total Earned: $${earned}</p>`
+  },
+
+  tripsTable: () => {
+    return `
+    <section class="table-container">
+      <table class="agent-table">
+        <tbody id="table">
+          <tr>
+           <th>Trip Id</th>
+           <th>Customer</th>
+           <th>Date of Travel</th>
+           <th>Status</th>
+           <th>Destination</th>
+          </tr>
+        </tbody>
+      <table>
+      <button class="show-all">Show All</button>
+    </section>
+    `
+  },
+
+  userListItem: (trip, user, destination) => {
```

Comment:
The formatting of html is a little off here.

Reply:
Thats odd that isn't how it looks on my atom